harvester/feature_enricher.py

```diff
@@ -152,7 +152,7 @@ def enrich(feature, feature_association):
 
 
         enriched_features = []
-        if len(description_parts[0].split('-')) == 2:
+        if len(description_parts[0].split('-')) == 2 and len(description_parts[0].split('-')[1]) > 1:
```

Comment:
This could be improved. For example, antisense genes (e.g. A1BG-AS1) would evaluate `True` despite being a single gene name. Instead, I propose we simply do a lookup from the table of approved HGNC gene symbols from [ftp://ftp.ebi.ac.uk/pub/databases/genenames/new/json/non_alt_loci_set.json](ftp://ftp.ebi.ac.uk/pub/databases/genenames/new/json/non_alt_loci_set.json). If when not split it matches a gene symbol, then it is not a fusion. Else, when split matches 2 gene symbols, it IS a fusion. Concurrently, we should enrich / normalize our "gene" concept with an Entrez ID from this file.

Reply:
>Instead, I propose we simply do a lookup from the table of approved HGNC gene symbols from ftp://ftp.ebi.ac.uk/pub/databases/genenames/new/json/non_alt_loci_set.json. If when not split it matches a gene symbol, then it is not a fusion.
Agreed.  I'll address in this PR

>Concurrently, we should enrich / normalize our "gene" concept with an Entrez ID from this file.
Agreed.  Can you open a new issue for this?  I'll address in a separate PR.

Reply:
Issue #102 created. Thanks @bwalsh.

Reply:
@ahwagner I've altered the fusion handling here.  We now use non_alt_loci_set.json.
Are we ready to merge?